docs/local_dev.md

```diff
@@ -314,3 +314,43 @@ sudo ./cli --install --os Ubuntu_20.04.1_x86-64 --bundle-repo 10.26.226.219:5000
 # Will override the OS detection, use the specified repo and uninstall
 sudo ./cli --uninstall --os Ubuntu_20.04.1_x86-64 --bundle-repo 10.26.226.219:5000/repo --k8s v1.22.3
 ```
+
+## Installer Implementation
```

Comment:
I think this deserves a separate page for anyone interested in writing their own installer?

The current page is dedicated to local development. The installer is something for the custom development of BYOH.

Reply:
Done. Created a separate doc for the installer and updated the root readme.